content/docs/command-reference/queue/start.md

```diff
@@ -0,0 +1,55 @@
+## queue start
+
+Start the
+[DVC experiments](/doc/user-guide/experiment-management/experiments-overview)
+task queue worker process.
```

Comment:
Let's use more user-perspective descriptions. In this case something like what we have for `exp run --run-all`:

"~~Run~~Start running all queued experiments ... outside your workspace (in `.dvc/tmp/exps`)."

Reply:
@pmrowla What do you think?